multi-int-store/build.gradle

```diff
@@ -8,6 +8,10 @@
 
 dependencies {
     implementation(Dependencies.springFrameworkSolr())
+    implementation("org.apache.zookeeper:zookeeper:3.5.5") {
```

Comment:
✏️ Should this be in it's own `Dependencies.zookeeper()`?

Reply:
Oh, thanks for the reminder.